static/build/miyoo/app/lang/pt-BR.lang

```diff
@@ -91,33 +91,33 @@
     "88": "ABRIR",
     "89": "VOLTAR",
     "90": "NAVEGAR",
-    "91": "Menu",
+    "91": "MENU",
     "92": "Continuar",
     "93": "Salvar",
     "94": "Carregar",
     "95": "Volume",
     "96": "Sair do jogo",
     "97": "Salvar %d/%d",
-    "98": "Vazia",
-    "99": "Sem imagens",
+    "98": "Vazio",
+    "99": "Sem capas",
     "100": "O carregamento falhou",
     "101": "O salvamento falhou",
-    "102": "slot %d salvo",
+    "102": "Espaço %d salvo",
     "103": "Tela cheia",
-    "104": "ON",
-    "105": "OFF",
+    "104": "ATIVO",
```

Comment:
Maybe it would make more sense as "ATIVADO" / "DESATIVADO"?

Reply:
I agree with that.